hieradata/site/dev/role/rke2agent.yaml

```diff
@@ -0,0 +1,3 @@
+---
+rke2::release_series: "1.31"
```

Comment:
this is set in the common and since there is no mayor version jump, this can be safely removed.
https://github.com/lsst-it/lsst-control/blob/ba7eef4416384e176d56273a38483a9988d171a9/hieradata/common.yaml#L582